examples/generic_connector_configs/README.md

```diff
@@ -122,6 +123,53 @@ The configuration file for the Docker Registry API can be found at
 
 ___
 
+### Dropbox API
+This example can be used to interact with
+[Dropbox's API](https://www.dropbox.com/developers/documentation/http/overview).
+
+The configuration file for the Dropbox API can be found at
+[dropbox_secretless.yml](./dropbox_secretless.yml).
+
+#### How to use this connector
+* Edit the supplied configuration to get your [
+Dropbox API token](https://www.dropbox.com/developers/apps) or
+[App key and App Secret](https://www.dropbox.com/developers/apps)
+* Run Secretless with the supplied configuration
+* Query the Dropbox API using
+`http_proxy=localhost:8081 curl {Dropbox Route}/{Request}`
+
+#### Example Usage
+<details>
+  <summary><b>Example setup to try this out locally</b></summary>
+  <ol>
```

Comment:
I think the content of this expandable section can be done using markdown instead of inline HTML. Looks like inline HTML has also been used in expandable sections for  other generic connector examples, so maybe this can all be changed in a separate commit?:
```
#### Example Usage
<details>
  <summary><b>Example setup to try this out locally:</b></summary>

  1. Get an OAuth token from the settings page of a
     [Dropbox application](https://www.dropbox.com/developers/apps)
  1. Store the token from your request in your local credential manager so
      that it may be retrieved in your `secretless.yml`
  1. Build and run Secretless locally
     ```
     ./bin/build_darwin
     ./dist/darwin/amd64/secretless-broker \
          -f examples/generic_connector_configs/dropbox_secretless.yml
     ```
  1. On another terminal window, make a request to Dropbox using Secretless
     ```
     http_proxy=localhost:8081 curl -X POST api.dropboxapi.com/2/team/get_info
     ```
</details>

```
A few notes:
* Need an empty line after the `<summary>...</summary>` line.
* Lines can all be number `1.`, markdown should take care of incrementing.